### PR TITLE
Fixes numberValue() bug which will fry on any bigints requiring over 53 bits.

### PR DESCRIPTION
**Pull request by wesboyt**
*Issue #, if available:*
https://github.com/amzn/ion-js/issues/420
*Description of changes:*
removes reliance on numberValue()

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/unit/IonDecimalTest.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ define([
             '12300d-2'  : () => test('12300d-2', '12300', -2, 123, '123.00'),
             '123456.0'  : () => test('123456.0', '1234560', -1, 123456.0, '123456.0'),
             '1234560d-1': () => test('1234560d-1', '1234560', -1, 123456.0, '123456.0'),
+            '9007199254740991.000000': () => test('9007199254740991.000000', '9007199254740991000000', -6, 9007199254740991, '9007199254740991.000000'),
```
Comment (zslayton) [inline tests/unit/IonDecimalTest.js]: Please add another test for an enormous negative number to also exercise the `if(cStr[0] === '-')` branch.

Comment (wesboyt) [reply tests/unit/IonDecimalTest.js]: Fasho im adding a few more edge case values as well.

Comment (wesboyt) [reply tests/unit/IonDecimalTest.js]: I believe this has been addressed.
